### PR TITLE
Check invalid handle native

### PR DESCRIPTION
**Pull request by CedricGuillemet**
To be used with https://github.com/BabylonJS/BabylonNative/pull/72

### CODE REVIEW

File: src/Engines/nativeEngine.ts [hunk 1]
```diff
@@ -71,8 +71,8 @@ interface INativeEngine {
     setFloat4(uniform: WebGLUniformLocation, x: number, y: number, z: number, w: number): void;
 
     createTexture(): WebGLTexture;
-    loadTexture(texture: WebGLTexture, buffer: ArrayBuffer | ArrayBufferView | Blob, mipMap: boolean, invertY: boolean): void;
-    loadCubeTexture(texture: WebGLTexture, data: Array<Array<ArrayBufferView>>, flipY : boolean): void;
+    loadTexture(texture: WebGLTexture, buffer: ArrayBuffer | ArrayBufferView | Blob, mipMap: boolean, invertY: boolean): any;
```
Comment (bghgary) [inline src/Engines/nativeEngine.ts]: Maybe return a book instead for these?

Comment (CedricGuillemet) [reply src/Engines/nativeEngine.ts]: What's a book?

Comment (bghgary) [reply src/Engines/nativeEngine.ts]: Aye, typing on my phone. Bool :)

Comment (CedricGuillemet) [reply src/Engines/nativeEngine.ts]: I'd like to be consistent with createIndexBuffer/createVertexBuffer

Comment (bghgary) [reply src/Engines/nativeEngine.ts]: I don't think that makes sense. The create functions create things and thus return the thing it created. The load functions load things into the first parameter :) A boolean return for load to indicate success makes the most sense to me.

**Pull request merged**